### PR TITLE
[SPARK-20026][Doc][SparkR] Add Tweedie example for SparkR in programming guide

### PR DESCRIPTION
**Pull request by actuaryzhang**
## What changes were proposed in this pull request?
Add Tweedie example for SparkR in programming guide. 
The doc was already updated in #17103.

### CODE REVIEW

**Comment by actuaryzhang** [comment]
@felixcheung

**Comment by felixcheung** [comment]
Could you add [SPARKR] to the PR title please

File: examples/src/main/r/ml/glm.R [hunk 1]
```diff
@@ -56,6 +56,15 @@ summary(binomialGLM)
 # Prediction
 binomialPredictions <- predict(binomialGLM, binomialTestDF)
 head(binomialPredictions)
+
+# Fit a generalized linear model of family "tweedie" with spark.glm
+training3 <- read.df("data/mllib/sample_multiclass_classification_data.txt", source = "libsvm")
+tweedieDF <- transform(training3, label= training3$label * exp(randn(10)))
```
Comment (felixcheung) [inline examples/src/main/r/ml/glm.R]: nite, style: `label = trai...`

**Review by felixcheung — CHANGES_REQUESTED** [review]
thanks, minor comment

**Comment by actuaryzhang** [comment]
Issues fixed. Thanks.

**Comment by felixcheung** [comment]
merged to master, thanks!

**Pull request closed without merging**